homeassistant/components/august/entity.py

```diff
@@ -0,0 +1,67 @@
+"""Base class for August entity."""
+
+import logging
+
+from homeassistant.core import callback
+
+from . import DEFAULT_NAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class AugustEntityMixin:
```

Comment:
This should inherit from `Entity` to make it clear that the mixin should mix with an `Entity` tree.

Reply:
Will adjust when I move more of the battery code out in favor or it being in py-august after https://github.com/snjoetw/py-august/pull/45 merges

Reply:
Ended up fixing it in https://github.com/home-assistant/home-assistant/pull/32340 as https://github.com/snjoetw/py-august/pull/45 is waiting for merge